gradle.properties

```diff
@@ -24,5 +24,5 @@ logbackVersion=1.3.14
 commonsIoVersion=2.16.1
 
 junitVersion=4.13.2
-powermockVersion=2.0.9
+mockitoVersion=5.12.0
```

Comment:
Switched from powermock to mockito due to powermock/jdk 11 issues e.g. the test `NexusIqScanTaskTest.testScan_realWithScanTargets` would print warnings for illegal reflective access. The approach taken with mockito does not.

Reply:
We were already using Mockito through `org.powermock:powermock-api-mockito2` so now using just Mockito, given that it supports mocking static methods, seems ok.